src/scope.ml

```diff
@@ -208,15 +245,24 @@ let scope : mode -> sig_state -> env -> p_term -> tbox = fun md ss env t ->
     in
     aux env xs
   (* Scoping function for head terms. *)
-  and scope_head : env -> p_term -> tbox = fun env t ->
+  and scope_head : ?allow_private:bool -> env -> p_term -> tbox =
+    fun ?allow_private env t ->
     match (t.elt, md) with
-    | (P_Type          , M_LHS(_) ) ->
+    | (P_Type          , M_LHS(_)         ) ->
```

Comment:
@rlepigre I don't like this idea of indenting the right parenthesis in matching clauses because it introduces spurious diffs in commits. I propose not to continue to do this.

Reply:
You can choose to not display space related changes via a tickbox in the github interface

Reply:
I would prefer to continue doing that, because this makes the code more readable. As Gabriel said, there are ways to ignore space changes when looking at diffs.

Reply:
I didn't know. Thanks!